src/utils/helpers/colors.ts

```diff
@@ -0,0 +1,26 @@
+import { KUSAMA } from "../constants/relay-chain-names";
+
+/**
+ * Helper to get status color variant compatible with theme. 
+ * @param { 'green' | 'yellow' | 'red'} color Color variant.
+ * @param {string} cssProperty CSS Property to prefix color with.
+ * @return {string} Appropriate color shade.
+ */
+const getColorShade = (color: 'green' | 'yellow' | 'red', cssProperty = 'text'): string => {
```

Comment:
The dApp was missing status color variations for Interlay theme, I decided to handle it with this function. This also makes maintainability much easier in case we want to change colors with the dApp redesign.

Reply:
I think we can use an enum instead of raw strings.

Reply:
> I think we can use an enum instead of raw strings.

I think the use of type union is more approppriate in this case to avoid unnecessary enum imports in affected components. Do you have strong opinion about why to use the enum? :)